pages/privacy-policy.js

```diff
@@ -46,7 +46,7 @@ const privacyPolicy = () => {
                         </div>
                     </div>
                     <p id="2" className={[style.modal__button__para, style.none].join(' ')}>
-                        We are more than aware of your private data. We have taken care of end to end encryption as much as we could have. All your passwords are hashed before going into our database. We however suggest you to not to use your common password on this website. You can use auto password gernerator on any browser for the same. Team DevBucket does not hold any guarantee on any possible leakage of private data or your music. We However assure you that we have followed all the best practices to secure your data
+                    We are more than aware of your private data. We have taken care of end-to-end encryption as much as we could have. All your passwords are hashed before going into our database. We however suggest you to not to use your common password on this website. You can use an auto password generator on any browser for the same. Team DevBucket does not hold any guarantee on any possible leakage of private data or your music. We However assure you that we have followed all the best practices to secure your data
                     </p>
```

Comment:
replace text with `At Vovoca, your privacy is our utmost priority. We understand the sensitivity of your personal information and have taken extensive measures to safeguard it. Through meticulous implementation of end-to-end encryption, we strive to ensure that your data remains protected throughout its journey. To provide an added layer of security, all passwords are securely hashed before being stored in our database. As an extra precaution, we strongly recommend refraining from using commonly used passwords on this platform. Instead, leverage the convenience of auto password generators available on various browsers. While Team DevBucket cannot provide an absolute guarantee against any potential data breaches or music leaks, we assure you that we have diligently adhered to industry best practices to fortify the security of your data. Rest assured, your privacy is in safe hands at Vovoca.`